documentation/docs/steps/checkChangeInDevelopment.md

```diff
@@ -6,6 +6,37 @@
 
 * No prerequisites
 
+**Note:** This step is deprecated. Use the [isChangeInDevelopment](isChangeInDevelopment.md) instead.
```

Comment:
This step has been deprecated. Use the step [isChangeInDevelopment](isChangeInDevelopment.md) instead.